provider/inspec.rb

```diff
@@ -161,5 +173,23 @@ def nested_object_requires(nested_object_type)
         [nested_object_type.__resource.name, nested_object_type.name.underscore].join('_')
       ).downcase
     end
+
+    def resource_name(object, product_ns)
+      "google_#{product_ns.downcase}_#{object.name.underscore}"
+    end
+
+    def sub_property_descriptions(property)
+      if nested_object?(property)
+        return property.properties.map \
+          { |prop| "    * `#{prop.name}`: #{prop.description}" }.join("\n")
+      end
+      # rubocop:disable Style/GuardClause
+      if typed_array?(property)
+        return property.item_type.properties.map \
+          { |prop| "    * `#{prop.name}`: #{prop.description}" }.join("\n")
+      end
+      # rubocop:enable Style/GuardClause
```

Comment:
In this case, I think we'd want a guard clause.

Reply:
Rubocop was contradicting itself. Complaining about guard clause, but then if I add a guard clause it says multi line guard clause should be regular if statement